app/scripts/components/common/map/style/button.tsx

```diff
@@ -0,0 +1,27 @@
+import styled from 'styled-components';
+import { createButtonStyles } from '@devseed-ui/button';
+import { themeVal } from '@devseed-ui/theme-provider';
+
+import { TipButton } from '$components/common/tip-button';
+
+// Why you ask? Very well:
+// Mapbox's css has an instruction that sets the hover color for buttons to
+// near black. The only way to override it is to increase the specificity and
+// we need the button functions to get the correct color.
+// The infamous instruction:
+// .mapboxgl-ctrl button:not(:disabled):hover {
+//   background-color: rgba(0, 0, 0, 0.05);
+// }
```

Comment:
❓ 💭 : Do we want this note here? If this is helpful, might be worth including the link to this documentation. Then this is also a random thought which we can discuss outside this PR... but I was thinking we should probably start marking our in-code documentation with maybe helpful pre-fixes (ex: `NOTE, COMMENT, TODO, TECH-DEBT`) this would help us identify helpful comments in the code, todo's to come back too and then tech-debt meant to be thought about and revisited? Again this is a discussion for a different time but wanted to say it here to start the convo, lmk if you have thoughts

Reply:
Yeah, having a prefix is a great idea. Is there any way to automatically check these prefixes for comments? I do not trust myself to remember it!

Reply:
I think we will just have to create that standard ourselves and document it somewhere for development purposes